src/pages/Home/component/Post/Post.tsx

```diff
@@ -13,7 +13,7 @@ const Post = (props: Props) => {
 	return (
 		<div className="post" key={post.id} onClick={() => onClick(post.id)}>
 			<p className="post__id">id: {post.id}</p>
-			<p className="post__title">title: {post.body}</p>
+			<p className="post__title">title: {post.title}</p>
```

Comment:
I don't understand why we need a title text here? Does the test fail if there is no "title" text here as well as in the test itself?

Reply:
The idea is that PostList has id and titile and when you click on it, you see the body text